### PR TITLE
OSDOCS-640: Limiting support for proxy to UPI installs

### PR DESCRIPTION
**Pull request by bergerhoffer**

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@openshift/team-documentation For peer review please.

File: release_notes/ocp-4-2-release-notes.adoc [hunk 1]
```diff
@@ -105,9 +105,11 @@ for details.
 
 {product-title} {product-version} introduces support for installing and updating
 an OpenShift cluster through a corporate proxy server. Proxy information
```
Comment (kalexand-rh) [inline release_notes/ocp-4-2-release-notes.adoc]: s/OpenShift/{product-title}
s/server./server on user-provisioned infrastructure.

**Comment by bergerhoffer** [comment]
@lihongan @gpei Does this update look okay to you?

**Comment by gpei** [comment]
The changes lgtm, thanks!

**Comment by lihongan** [comment]
LGTM, thanks

**Pull request merged**